src/InterfaceConstants.h

```diff
@@ -73,4 +73,8 @@
 // Shared region polygon approximation
 #define DEFAULT_VERTEX_COUNT 1000
 
+// socket port
+#define DEFAULT_SOCKET_PORT 3002
```

Comment:
@pford we originally wanted this file for ICD-related constants. Should we move the other `#defines` out of this file to a separate file?

Reply:
I would rather rename the file and keep them in one place.

Reply:
It looks like we no longer use the input parameter "threads",  TBB_THREAD_COUNT (its default value), or its variable thread_count in Main.cc.

Reply:
@markccchiang thanks for cleaning this up!